mempool/bench_test.go

```diff
@@ -50,3 +98,34 @@ func BenchmarkCacheRemoveTime(b *testing.B) {
 		cache.Remove(txs[i])
 	}
 }
+
+// BenchmarkClistMempoolGetNextTxBytes-8   	       2	 793475198 ns/op
+func BenchmarkClistMempoolGetNextTxBytes(b *testing.B) {
+	benchmarkMempoolGetNextTxBytes(b, enumclistmempool)
+}
+
+// BenchmarkLLRBMempoolGetNextTxBytes-8   	     322	   3565640 ns/op
+func BenchmarkLLRBMempoolGetNextTxBytes(b *testing.B) {
+	benchmarkMempoolGetNextTxBytes(b, enumllrbmempool)
+}
+
+func benchmarkMempoolGetNextTxBytes(b *testing.B, enum mpEnum) {
+	app := kvstore.NewApplication()
+	cc := proxy.NewLegacyLocalClientCreator(app)
+	mempool, cleanup := newLegacyMempoolWithAppAndConfig(cc, cfg.ResetTestRoot("mempool_test"), enum)
+	defer cleanup()
+	size := 100000
+	for i := 0; i < size; i++ {
+		rand, _ := rand.Int(rand.Reader, big.NewInt(100))
+		tx := "k" + strconv.Itoa(i) + "=v" + strconv.Itoa(i) + ",f," + strconv.FormatInt(rand.Int64(), 10)
```

Comment:
I thought we should do modulo on priorities?

Reply:
Using crypto rand to pass lint here, already ranged in [1,100)
```
rand, _ := rand.Int(rand.Reader, big.NewInt(100))
```